servatrice/servatrice.ini.example

```diff
@@ -99,6 +99,15 @@ allowedpunctuation=_.-
 ; If a username can begin with punctuation defined in allowedpunctuation
 allowpunctuationprefix=false
 
+; Enable forgot password functionality
+enableForgotPassword=true
+
+; Validate users inputed email matches email on account before allowing forgot password reset
+forgotpasswordemailrequired=true
```

Comment:
Sure.  I couldnt decide if it should be located under the users's section of the ini or if we should make a new section for it?

Thoughts?